src/Halogen/HTML/Properties.purs

```diff
@@ -2,38 +2,45 @@
 module Halogen.HTML.Properties
   ( key
   , alt
+
   , charset
-  , class_
-  , classes
+  , class_, classes
   , cols
   , rows
   , colSpan
   , rowSpan
   , for
   , height
+  , width
   , href
   , id_
   , name
   , rel
   , src
   , target
   , title
+
   , type_
+
   , value
-  , width
   , disabled
+  , enabled
+
   , required
   , readonly
   , spellcheck
-  , enabled
   , checked
   , selected
   , placeholder
   , autocomplete
   , autofocus
   , multiple
+
   , draggable
+  , tabIndex
+
   , ref
+
```

Comment:
This diff noise is because I matched up the spacing and organisation with the list in `Indexed` so it's easier to see the differences side by side.